tools/templates/extras/const.jinja2

```diff
@@ -1,8 +1,10 @@
-def const(
+def _const(
```

Comment:
The only value it is meant to have is having a legacy version if someone _really_ needs it. I can drop it if we don't need to ease the transition further.

Reply:
As in: The user **really** wants to use protobuf's `value_float` field instead of anything else? If somebody wants to use `float32` in general they just have to specify the `dtype` and all is well. I think we can remove this.